QUEST.tsv

```diff
@@ -498,20 +498,20 @@ QUEST_20150317_000497	The harder you try to find this, the stronger you become,
 QUEST_20150317_000498	Aide de camp Selina
 QUEST_20150317_000499	Our days to stay self-sufficient is growing. {nl}We even joke about plowing a farm ourselves,  {nl}but it's not becoming a joke anymore.
 QUEST_20150317_000500	Watcher
-QUEST_20150317_000501	If it isn't the one who made me release the Saule goddess. {Nl}What are you putting your life on the line for?
-QUEST_20150317_000502	There is no need for more of your work. {nl}You should follow the others by immersing yourself in peace {nl}and praying the Goddess will grant you your wishes.
-QUEST_20150317_000503	I warned you, but you just won't listen.
+QUEST_20150317_000501	If it isn't you, the one who made me release the Saule Goddess. {nl}Then what are you putting your life on the line for?
```

Comment:
If it isn't you, the one who made me release the Saule Goddess

↓

If it isn't you, the one who made me release Goddess Saule. 

Sounds better?